ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -103,10 +103,9 @@ def decelerate_waypoints(self, waypoints, closest_idx):
             stop_idx = max(self.stopline_wp_idx - closest_idx - 2, 0) # Two waypoints back from so front of car stops at line
             dist = self.distance(waypoints, i, stop_idx)
             vel = math.sqrt(2 * MAX_DECEL * dist)
-
+            vel = 0.02 * MAX_DECEL * dist
```

Comment:
if you write this way, the above line will never be useful.
can this line causes the car moves really slowly at the beginning.